src/main/webapp/app/pages/curation/collapsible/MutationCollapsible.tsx

```diff
@@ -81,27 +82,30 @@ const MutationCollapsible = ({
 
   const [mutationUuid, setMutationUuid] = useState<string>('');
   const [mutationName, setMutationName] = useState<string>('');
-  const [mutationNameReview, setMutationNameReview] = useState<Review>(null);
-  const [mutationAlterations, setMutationAlterations] = useState<Alteration[]>(null);
+  const [mutationNameReview, setMutationNameReview] = useState<Review | null>(null);
+  const [mutationAlterations, setMutationAlterations] = useState<Alteration[] | null>(null);
   const [isRemovableWithoutReview, setIsRemovableWithoutReview] = useState(false);
   const [relatedAnnotationResult, setRelatedAnnotationResult] = useState<AlterationAnnotationStatus[]>([]);
 
   useEffect(() => {
-    setRelatedAnnotationResult(annotatedAltsCache.get(hugoSymbol, [{ name: mutationName, alterations: mutationAlterations }]));
-  }, [annotatedAltsCache.loading, mutationName, mutationAlterations]);
+    const arr = annotatedAltsCache?.get(hugoSymbol ?? '', [{ name: mutationName, alterations: mutationAlterations }]) ?? [];
+    setRelatedAnnotationResult(arr.filter((x): x is AlterationAnnotationStatus => x !== undefined && x !== null));
+  }, [annotatedAltsCache?.loading, mutationName, mutationAlterations]);
 
   const exons = useMemo(() => {
     return _.uniqBy(
       relatedAnnotationResult.reduce((acc, next) => {
-        acc.push(...next.annotation.exons);
+        if (next.annotation?.exons) {
+          acc.push(...next.annotation.exons);
+        }
         return acc;
       }, [] as ProteinExonDTO[]),
       'exon',
-    ).sort((a, b) => a.range.start - b.range.start);
+    ).sort((a, b) => (a.range?.start ?? 0) - (b.range?.start ?? 0));
```

Comment:
Instead of `0`, do we want to use a large positive value? That way, items with no range will go afters exon with a range.

Reply:
I agree with Ben.